### ISSUE TITLE
[BUG] Repeater field won't show unless there's also an 'html' field

### ISSUE THREAD

**Issue by smokeysmokeyd** [bolt/bolt#4953]
Using the Bolt Master version I am unable to add a repeater field to a content-type unless that content-type has an 'html' field.

This bug puzzled me for a while because the install includes a 'showcase' type which displays the repeater field, and works fine. But, for any of my content-types I tried to write myself the repeater didn't work. Eventually I cloned the 'showcases' content-type and whittled down at the components to figure out what was causing it. The only way I could get the repeater to show is if I had an `html: type: html` field.

Details
-------

 - Relevant Bolt Version: [ master ]
 - Install type: [ Composer install ]
 - PHP version: [ 5.5.9  ]
 - Used webserver: [ Apache ] version [ 2.4.7 ]
 - For UX/UI issues: [ 46.0.2490.80 Ubuntu ]

Reproduction
------------

(1) New fresh install of Bolt Master

(2) Add the following content-types to content-types.yaml
```
# Repeater field w/o HTML field
test1:
    name: test1
    slug: test1
    singular_name: tes1
    singular_slug: tes1
    fields:
        repeater:
            type: repeater
            limit: 3
            prefix: "<p>This allows you to create multiple sets of fields. Use the add button at the bottom to create a new empty set.</p>"
            fields:
                test1title:
                    type: text
                test1image:
                    type: image
                    extensions: [ gif, jpg, png ]
                test1content:
                    type: html

# Repeater field w/ HTML field
test2:
    name: test2
    slug: test2
    singular_name: tes2
    singular_slug: tes2
    fields:
        html:
            type: html
            height: 150px
        repeater:
            type: repeater
            limit: 3
            prefix: "<p>This allows you to create multiple sets of fields. Use the add button at the bottom to create a new empty set.</p>"
            fields:
                test1title:
                    type: text
                test1image:
                    type: image
                    extensions: [ gif, jpg, png ]
                test1content:
                    type: html
```

(3) For me, this results in the following for `test1`:
![image](https://cloud.githubusercontent.com/assets/4089371/13518952/43cb08d0-e190-11e5-9a63-9c346b454bb5.png)

And for `test2`:
![image](https://cloud.githubusercontent.com/assets/4089371/13518974/640a31fc-e190-11e5-8d13-aa19dea8e9c9.png)

(4) As you can see, on `test1` is the error in the console:
```
bolt.js:4 Uncaught TypeError: Cannot read property 'replace' of undefined
```

I'd be happy to investigate if you could point me in the direction of where the error could be in the code? Seems bizarre everything works with an HTML field but no other way.

Thanks so much!
Smokey

**Comment by rarila** [comment]
@smokeysmokeyd Thanks, good catch! Bug is found, patch is on the way.

**Comment by GawainLynch** [comment]
Fixed in #4956
